### DISCUSSION TITLE
Default mode for Validation message

### DISCUSSION THREAD

**Discussion by petrkasnal** [havit/Havit.Blazor#403] [Q&A] [answered]
Hello, 

i use your inputs and use your validations - https://havit.blazor.eu/components/HxValidationMessage. I would like set another default mode. What's best option place it? Startup, App, MainLayout? 

Thank you

**Comment by crdo** [comment, marked as answer]
Hi @petrkasnal, best place to set Defaults is usually Startup.cs or Program.cs as per the documentation https://havit.blazor.eu/concepts/defaults-and-settings

**Reply by petrkasnal** [reply]
Thank you very much
